lib/source.c

```diff
@@ -109,6 +109,15 @@ tlog_source_read(struct tlog_source *source, struct tlog_pkt *pkt)
 
     assert(grc == TLOG_RC_OK || tlog_pkt_is_void(pkt));
     assert(tlog_source_is_valid(source));
+#ifndef NDEBUG
+#include <tlog/timespec.h>
+#include <tlog/delay.h>
+    struct timespec diff;
```

Comment:
Please get those into their places too. Thanks!

Reply:
Do you mean put these line in Header file and including `diff` variable?

Reply:
Just imagine where you would put them *without* `#ifndef NDEBUG` (forget about it for a moment), and where you in fact already put them before, then simply wrap them in that `#ifndef` right in place.